mypy/test/helpers.py

```diff
@@ -3,14 +3,18 @@
 import sys
 import time
 
-from typing import List, Dict, Tuple, Callable, Any
+from typing import List, Dict, Tuple, Callable, Any, Optional
 
 from mypy import defaults
-from mypy.myunit import AssertionFailure
+
+import pytest  # type: ignore  # no pytest in typeshed
+from unittest import TestCase as Suite
```

Comment:
Add comment about this, since this may be kind of unexpected. In the future we'd probably want to directly import `TestCase` in test files and inherit from it instead of the `Suite` alias (and maybe rename some of the other helper classes to avoid ambiguity).